sources/packages/backend/apps/api/src/route-controllers/student/_tests_/e2e/student.external.controller.searchStudent.e2e-spec.ts

```diff
@@ -352,6 +493,271 @@ describe("StudentExternalController(e2e)-searchStudents", () => {
     },
   );
 
+  it(
+    "Should return student and application information from SFAS when the student with provided SIN exist in SFAS" +
+      ` and does not exist in SIMS and the student has ${OfferingIntensity.fullTime} applications in current program year` +
+      " ignoring the applications from old program years and the student does has one or more dependants.",
```

Comment:
When it says "ignoring the applications from old program years" is it expected that some application would be associated with a past program year in the data setup and then ignored?